### DISCUSSION TITLE
Are caches limited to one repository?

### DISCUSSION THREAD

**Discussion by evancampbell** [actions/cache#691] [Q&A]
By that I mean, can I save to a cache in one repository, then load it from a different repository?

**Comment by bishal-pdMSFT** [comment]
Yes, cache are limited to one repository (and forked repository as well). In fact they are more restricted based on the branch and the key. Please see [here](https://docs.github.com/en/actions/advanced-guides/caching-dependencies-to-speed-up-workflows#restrictions-for-accessing-a-cache) and [here](https://github.com/actions/cache#cache-scopes) for more details.

Are you having an use case which is not satisfied by this pattern?
